python/ray/tune/durable_trainable.py

```diff
@@ -22,13 +22,14 @@ class DurableTrainable(Trainable):
 
     Attributes:
         remote_checkpoint_dir (str):  Upload directory (S3 or GS path).
+            If None, this class acts like a Trainable class.
         storage_client: Tune-internal interface for interacting with external
             storage.
 
     >>> tune.run(MyDurableTrainable, sync_to_driver=False)
     """
 
-    def __init__(self, remote_checkpoint_dir, *args, **kwargs):
+    def __init__(self, remote_checkpoint_dir=None, *args, **kwargs):
```

Comment:
oh hmm maybe expand **kwargs to the actual variables and get rid of *args. 

i think what you have probably works in this case since trainable doesn't have any positionals but otherwise this would be a syntax error